### DISCUSSION TITLE
Building UTM.HV.ipa

### DISCUSSION THREAD

**Discussion by jnawaz** [utmapp/UTM#5107] [UTM Help (iOS)]
Is there a particular build script command to build a hypervisor enabled ipa for the M1 ipad as per the [documentation](https://docs.getutm.app/installation/ios/#:~:text=app/%20for%20UTM.-,Summary,-File)?

**Comment by osy** [comment]
Yeah sorry it hasn’t been updated but the package.sh script supports the option to create the hv ipa
